src/main/java/com/internet/shop/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -18,6 +19,26 @@ public ShoppingCart create(ShoppingCart shoppingCart) {
         return shoppingCartDao.create(shoppingCart);
     }
 
+    @Override
+    public ShoppingCart get(Long id) {
+        return shoppingCartDao.get(id).get();
+    }
+
+    @Override
+    public List<ShoppingCart> getAll() {
+        return shoppingCartDao.getAll();
+    }
+
+    @Override
+    public ShoppingCart update(ShoppingCart item) {
```

Comment:
```suggestion
    public ShoppingCart update(ShoppingCart shoppingCart) {
```
same with other methods

Reply:
done